src/client/Directory.Build.props

```diff
@@ -35,6 +35,7 @@
     <Authors>Microsoft</Authors>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=844761</PackageProjectUrl>
+    <PackageReleaseNotes>A changelog is available at https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/blob/dev/changelog.txt </PackageReleaseNotes>
```

Comment:
Thanks @bgavrilMS this is an awesome improvement.
I wonder if we should not add 
```
and the roadmap (including blog post for releases) from https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki#roadmap
```

Reply:
Ok, but let's exclude the parenthesis - too many words for such a small place.